### PR TITLE
Iq interpolation params

### PR DESCRIPTION
**Pull request by brikendr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,6 +48,9 @@
     "publish-package": "npm run build && npm run build-web && npm publish",
     "compodoc": "./node_modules/.bin/compodoc -w -p tsconfig.json -s -n 'huddly-sdk' -d 'docs' --minimal --customFavicon 'docs_style/assets/imgs/favicon.ico' --disableGraph --disableInternal --hideGenerator --includesName 'Examples' --includes examples -a docs_style/assets -y docs_style"
   },
+  "pre-commit": [
```
[the remark this answers is not in the document]
Comment (brikendr) [reply package.json]: Yup! Pretty neat.

**Comment by brikendr** [comment]
@jontore i think we can merge this PR since the last camera software release supports setting/getting interpolation parameters!

**Comment by jontore** [comment]
@brikendr  What is the status here?

**Comment by brikendr** [comment]
@jontore it is ready for merge!

**Pull request merged**